source/libraries/Configuration.ts

```diff
@@ -13,8 +13,9 @@ export class Configuration {
     public readonly contractInterfacesOutputPath: string;
     public readonly controllerAddress: string|undefined;
     public readonly createGenesisUniverse: boolean;
+    public readonly useControlledTime: boolean;
 
-    public constructor(host: string, port: number, gasPrice: BN, privateKey: string, contractSourceRoot: string, contractOutputRoot: string, controllerAddress: string|undefined, createGenesisUniverse: boolean=true) {
+    public constructor(host: string, port: number, gasPrice: BN, privateKey: string, contractSourceRoot: string, contractOutputRoot: string, controllerAddress: string|undefined, createGenesisUniverse: boolean=true, useControlledTime: boolean=true) {
```

Comment:
We should probably change this flag to `isProduction` or something since there are now two things different about a deploy to production vs testnet:
1. `Time` vs `TimeControlled`
2. `LegacyReputationToken` vs `0xE94327D07Fc17907b4DB788E5aDf2ed424adDff6`

I suspect before launch there will be more things like this, though my hope is that we can keep the number as small as possible.

Reply:
Yea good call, will do